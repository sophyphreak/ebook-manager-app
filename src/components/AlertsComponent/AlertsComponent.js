import React from 'react';
import {
  Col,
  Row
} from 'reactstrap';

import AlertsPage from "./AlertsPage/AlertsPage";
import AlertsSuccess from './AlertsPage/AlertsSuccess';

const AlertsComponent = ({
  currentPage,
  ...props
}) => (
  <Row>
    <Col xs="12" sm={{ size: 8, offset: 2 }}>
      {
        currentPage === "AlertsPage" && (
          <AlertsPage {...props} />
        )
      }
      {
        currentPage === "SubmissionSuccess" && (
          <AlertsSuccess feature="alert" />
        )
      }
    </Col>
  </Row>
);

export default AlertsComponent;