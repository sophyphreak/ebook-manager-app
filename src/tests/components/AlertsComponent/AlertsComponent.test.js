import React from 'react';
import { shallow } from 'enzyme';
import AlertsComponent from '../../../components/AlertsComponent/AlertsComponent';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<AlertsComponent />);
});

test('should render AlertsComponent correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
