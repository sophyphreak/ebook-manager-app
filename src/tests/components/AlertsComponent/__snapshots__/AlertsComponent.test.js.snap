// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AlertsComponent correctly 1`] = `
<Row
  tag="div"
>
  <Col
    sm="3"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  />
  <Col
    sm="6"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs="12"
  />
</Row>
`;
